packages/shared/components/DelegationActivityDetails.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+    import { AddressBox, ActivityInclusionStatusPill, AmountBox } from '@ui'
+    import { DelegationActivity, getAssetFromPersistedAssets } from '@core/wallet'
+    import { getCoinType } from '@core/profile'
+
+    export let activity: DelegationActivity
+
+    const asset = getAssetFromPersistedAssets(getCoinType())
+    $: amount = Number(activity?.delegatedAmount)
```

Comment:
```suggestion
    $: amount = activity.delegatedAmount
```

Reply:
I also think that `AmountBox` should accept a bigint instead of a number but that's not an issue of this PR